app/assets/stylesheets/custom.scss

```diff
@@ -153,6 +153,10 @@ td {
   @include box-sizing;
 }
 
+#complete_your_loan {
```

Comment:
Not sure why hound doesn't like id selectors.  I think this is a good case to use it.  I'll update the id to use hyphens.